app-modules/billing/src/Stripe/Subscription/StripeAdapter.php

```
@@ -0,0 +1,102 @@
+        $session = $builder
+            ->withMetadata($data->metadata)
+            ->checkout([
+                'success_url' => $data->successUrl,
+                'cancel_url' => $data->cancelUrl,
+                'customer_update' => ['address' => 'auto'],
+            ]);
+
+        return $session->asStripeCheckoutSession()->url;
+    }
+
+    public function getBillingPortalUrl(Company|User $billable, string $returnUrl, array $options = []): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$options'. (undefined)

(UnusedFormalParameter)